vercel.json

```diff
@@ -1,14 +1,173 @@
 {
   "version": 2,
-  "routes": [
+  "redirects": [
     {
-      "src": "/posts/(.*).html",
-      "dest": "/posts/$1"
+      "source": "/",
+      "destination": "https://checklyhq.com/learning/headless"
     },
     {
-      "src": "/assets/(.*)",
-      "headers": { "cache-control": "max-age=315360000, s-maxage=315360000" },
-      "dest": "/assets/$1"
+      "source": "/posts/",
+      "destination": "https://checklyhq.com/learning/headless"
+    },
+    {
+      "source": "/posts/basics-puppeteer-intro/",
+      "destination": "https://checklyhq.com/learning/headless/<replace>/basics-puppeteer-intro/"
```

Comment:
What is `<replace>`?

Reply:
Can you please redirect to **www**.checklyhq.com instead of just checklyhq.com

Reply:
This is a draft, just wanted to validate with you the approach for redirection. 
- <replace> is a placeholder, need to change that with final path in HD
- Will change to www.*

Reply:
Cool! I can confirm that a 308 redirect works.